scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -64,13 +64,22 @@
                             {% endif %}
                         },
                       {% endfor %}
-                      {% for custTrack in cloud_public_tracks %}
+                      {% for custTrack in config_custom_tracks %}
```

Comment:
Renamed cloud_public_tracks -> **config**_custom_tracks, because it's not obvious that the tracks are in the cloud and it's easier to understand that they are specified in the config file